src/main.rs

```diff
@@ -173,6 +175,23 @@ fn validate_byte_size(s: String) -> Result<(), String> {
         |_| format!("{:?} is not a byte size", s))
 }
 
+// Create a regular expression from a file pattern:
+// Change
+// 1. "." to "\\."
+// 2. "%" to "."
+// 3. "*" to ".*"
+fn regexp_from_filepat(s: &str) -> Regex {
+    let string = s.to_string().replace(".","\\.").replace("%",".").replace("*",".*");
+
+    Regex::new(format!(r#"{}"#, string).as_str()).unwrap()
+}
+
+fn make_regexp(s: &str) -> Regex {
+    //    let es : String = regex::escape(s);
+    let rs = format!(r#"{}"#, s);
+    Regex::new(rs.as_str()).expect("Invalid regex given")
```

Comment:
Invalid regex shouldn't result in a panic, it should be a controlled exit.

Reply:
Yes, agree